posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+fast I/O external hard drive may also be an option. 
+:::
+
+
+
+### 1. Export Raw Data {#sec-export-raw-data}
+
+In AtoMx SIP, in the `Study details` panel on the upper left, click `Export` (@fig-atomx-export1). 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-atomx-export1"
+#| fig-cap: "Click `Export` (available in AtoMx SIP version 1.3+)."
+
+knitr::include_graphics("./figures/export-fig1.png")
+```
+
+Configure your export with the options indicated in @fig-atomx-export2 and click `EXPORT`. If you
+would like to view metadata (optional). You can go ahead and download the Seurat data
+now or as a separate step (_i.e._, Seurat data is not needed for stitching napari
+files). 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-atomx-export2"
+#| fig-cap: "Example configuration for exporting. For more information on extracting cell-level metadata, see @sec-adding-metadata."
+
+knitr::include_graphics("./figures/export-fig2.png")
+```
+
+When the export is ready, download the data. You can do this over the sftp 
+protocol in a variety of application. Here, I'm using [Cyberduck](https://cyberduck.io/){target="_blank"} but you can use other programs. 
+
+In Cyberduck, click `Open Connection`. In the dropdown menu, select `SFTP` and 
+enter the URL, username, and your (AtoMx) password. Then click `Connect`. Example:
+@fig-atomx-export3.
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-atomx-export3"
+#| fig-cap: "Example configuration for Cyberduck SFTP."
+
+knitr::include_graphics("./figures/export-fig3.png")
+```
+
+Once connected, find the relevant folder, right click, and select `Download As...` (@fig-atomx-export4). 
+Choose the location to on your computer to store the data. You may be able to store
+it on a networked drive but this is currently untested. 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-atomx-export4"
+#| fig-cap: "Download raw data somewhere on your desktop."
+
+knitr::include_graphics("./figures/export-fig4.png")
+```
+
+### 2. Launch `napari-cosmx` {#sec-launch-napari-cosmx}
+
+In order to use the stitching widget in the plugin, we must first launch the 
+plugin. Currently, the only way to do that is to load an existing napari-ready 
+folder. This can be any napari slide (_e.g._, a previous study or the two-FOV 
+example described in @sec-napari-ready-files).
+
+### 3. Stitching images {#sec-stitch-images}
+
+Once `napari-cosmx` is launched, the stitching widget is located on the right-hand
+panel (@fig-stitch-widget1). 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-stitch-widget1"
+#| fig-cap: "Example showing a launched `napari-cosmx` plugin with a possibly-unrelated slide. To stitch, a new slide, Click `Browse...` in the `Stitch Images` widget located on the right-hand side of napari."
+
+knitr::include_graphics("./figures/stitch-widget1.png")
+```
+
+We need to tell `napari-cosmx` where the raw data is located. In the right-hand panel
```

Comment:
where the raw data _are_ located

Reply:
Ah! Thank you. 208857d88eb7bd81da7fd043536a8aad6f51e28b